### PR TITLE
[FIX] Notify admin was generating errors when Rocket.Cat user was edited or deleted

### PR DESCRIPTION
**Pull request by rodrigok**

### CODE REVIEW

**Comment by wreiske** [comment]
<3

**Comment by wreiske** [comment]
I wish there was a way to say "this is the super admin, don't message all admins!" We have several users set up as "admin" in the system that do NOT need to see any of these messages. Same goes for email notifications on new user activations, etc. Maybe a "super-admin" group is needed, or a preference somewhere to choose the default admin user that should receive all admin related notifications. If that field is filled in, only notify that user. Otherwise, notify all admins.

File: server/lib/sendMessagesToAdmins.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export function sendMessagesToAdmins({
+	fromId = 'rocket.cat',
+	checkFrom = true,
+	msgs = [],
+	banners = [],
+}) {
+	const fromUser = checkFrom ? Users.findOneById(fromId, { fields: { _id: 1 } }) : true;
+
+	Roles.findUsersInRole('admin').forEach((adminUser) => {
+		if (fromUser) {
```
Comment (sampaiodiego) [inline server/lib/sendMessagesToAdmins.js]: does this mean?
```
if (!fromUser) {
  return;
}
```

if so, if can be done outside the `forEach` loop as well

Comment (rodrigok) [reply server/lib/sendMessagesToAdmins.js]: The banner insertion doesn't depend on fromUser

**Pull request merged**